RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -174,6 +195,82 @@ string useItems(item it1, item it2)
 	return useItems(it1, it2, true);
 }
 
+boolean isSniffed(monster enemy, skill sk)
+{
+	//checks if the monster enemy is currently sniffed using the specific skill sk
+	boolean retval = false;
+	switch(sk)
+	{
+	case $skill[Transcendent Olfaction]:
```

Comment:
tab everything between the switch brackets one more for formatting

Reply:
oddly enough what I wrote is the format we use in most switches in autoscend. even though it looks weird

Reply:
done